src/app/core/services/personal-cards.service.ts

```
@@ -124,24 +201,55 @@
     return this.spenderPlatformV1ApiService
-      .post<PlatformApiResponse<PersonalCardPlatform>>('/personal_cards/delete', payload)
+      .post<PlatformApiResponse<PlatformPersonalCard>>('/personal_cards/delete', payload)
       .pipe(map((response) => response.data));
   }
 
-  deleteAccount(accountId: string, usePlatformApi: boolean): Observable<PersonalCard | PersonalCardPlatform> {
+  deleteAccount(accountId: string, usePlatformApi: boolean): Observable<PersonalCard | PlatformPersonalCard> {
     if (usePlatformApi) {
       return this.deleteAccountPlatform(accountId);
     }
     return this.expenseAggregationService.delete('/bank_accounts/' + accountId) as Observable<PersonalCard>;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)